### PR TITLE
Adding enclave app that hosts Echo service in a VM

### PR DESCRIPTION
**Pull request by dzmitry-huba**

### CODE REVIEW

File: quirk_echo_enclave_app/src/main.rs [hunk 1]
```diff
@@ -0,0 +1,61 @@
+//
+// Copyright 2022 The Project Oak Authors
```
Comment (stanischikn) [inline quirk_echo_enclave_app/src/main.rs:2]: 2023

File: quirk_echo_enclave_app/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+[package]
+name = "quirk_echo_enclave_app"
+version = "0.1.0"
+authors = ["Dzmitry Huba <huba@google.com>"]
+edition = "2021"
```
Comment (stanischikn) [inline quirk_echo_enclave_app/Cargo.toml:5]: Is this supposed to be the year?

File: quirk_echo_enclave_app/build.rs [hunk 3]
```diff
@@ -0,0 +1,19 @@
+//
+// Copyright 2022 The Project Oak Authors
```
Comment (stanischikn) [inline quirk_echo_enclave_app/build.rs:2]:
```suggestion
// Copyright 2023 The Project Oak Authors
```

File: quirk_echo_enclave_app/src/main.rs (hunk 1 above)
Comment (stanischikn) [inline quirk_echo_enclave_app/src/main.rs:2]:
```suggestion
// Copyright 2023 The Project Oak Authors
```

**Review by stanischikn — COMMENTED** [review]
LGTM

File: quirk_echo_enclave_app/Cargo.toml (hunk 2 above)
Comment (dzmitry-huba) [reply quirk_echo_enclave_app/Cargo.toml:5]: Yes, however this is not the current year, but rather the edition of the tool chain.

**Review by stanischikn — APPROVED** [review]

**Comment by rbehjati** [comment]
Cool! 🎉

**Pull request merged**